### ISSUE TITLE
GANs L4.3 Learning Rate Quiz Possible Bug

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/dlnd-issue-reports#281]
The quiz lecture opened for me after I finished the video on 4.2 

The first part of the quiz has 3 options. One of the options was already selected for me. Not sure how that happened, honestly. Thought of pointing it out in case it's a bug.

Also, this entire lesson on Hyperparameters was already introduced in the RNN section, so not sure why it's being repeated and being publicized as new content.

**Comment by sahiljuneja** [comment]
Okay, so for the first problem I mention above, I went to a different lecture and returned to the quiz, and that option was unclicked.

**Comment by clapollo** [comment]
I haven't been able to reproduce this issue (where something starts out already selected). 

As for the content, it *is* new content, but we added it in two places. We want it to appear earlier for new students, but didn't want anyone to miss it, so we added it to the current week for both cohorts. That put it in the RNN and GAN sections for now. Eventually we'll decide on one place for it to live.
